### ISSUE TITLE
Cms pages does not exist

### ISSUE THREAD

**Issue by grafiqer** [faktiva/prestashop-clean-urls#96]
This module it works great, thanks. But all cms pages "This page does not exist" error.
Simple: www.sarsana.com/bilgi/hakkimizda

**Comment by drAlberT** [comment]
More info plz

**Comment by kttmnn** [comment]
hi,

thank you for your fast reply and your help. 

`HTTP/1.1 404 Not Found
Date: Tue, 15 May 2018 13:46:42 GMT
Server: Apache
P3P: CP="IDC DSP COR CURa ADMa OUR IND PHY ONL COM STA"
Powered-By: PrestaShop
Set-Cookie: PrestaShop-ef2386e7cc4efc34e6dbbec84bd8044b=irJscaAholAAPnxLJGy9OXu0r9T0ei%2FdDe3BfYYtT7kyYR3pMkmGOxID09ao4PRgi660m0ToK8joOGFK9m995bxWDmDwTBGIXR10f4BCDz2x2SR%2FIVJT%2BBkfmjwxySmk8ZCVNHh3a2ix4LTvfDWE0QnNRZHsrJNMxc7iFpuxzOs%3D000115; expires=Mon, 04-Jun-2018 13:46:42 GMT; Max-Age=1727999; path=/; domain=p459350.mittwaldserver.info; secure; HttpOnly
Upgrade: h2,h2c
Connection: Upgrade
Content-Type: text/html; charset=utf-8`


Is that what you need?

**Comment by drAlberT** [comment]
Uhm this looks to be a 404 ..

**Comment by kttmnn** [comment]
Yes, but Prestashop doesn't show the 404 default page. I see a normal CMS page, but where the content should be, there is the message "This page does not exist". So the system thinks it's a CMS site. However, the content is not output.
